examples/tests/cases/case_av_example.yaml

```diff
@@ -48,7 +48,6 @@ TestBrowserSafariAV:
     - Role: desktopSafari
       Capabilities:
         safariOptions:
-          prefs:
```

Comment:
Was this change intentional? Does it not work if we use the splat operator when building the driver, like for other browsers?

Reply:
in the docs: https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/Safari/Options

there is not a constructor, so, I can pass it, but it will be useless... it might be from the parent Options, but I think right now is not implemented in any meaningful way.

Reply:
Thanks for the link. I agree that it seems there is no implementation for passing arguments, but I think it should still be fine to use the same splat for the Safari driver (unless it breaks something). All other browser constructors extend the parent anyway, so there shouldn't be much of a difference.
Do keep the TODO comment though :)

Reply:
Done, should be all good now